### ISSUE TITLE
Installation command doesn't work

### ISSUE THREAD

**Issue by ricardomatias** [pmndrs/react-three-next#120]
Whatever combination of commands that I use such as `npx create-r3f-app next my-app tailwind -ts`, it always returns:

```
Welcome. Project's generation started using create-R3F-App 🐱

  —— New update available, run the following npx create-r3f-app <next> my-app <tailwind|styled>? -ts
```

I'm using Node v18.15

**Comment by RoboticWater** [comment]
You can try running the yarn command `yarn create r3f-app next my-app -ts`. I was able to get that to install; however, I'm getting a fatal error during runtime (see: #125), which may suggest a deeper issue with Typescript here.

**Comment by RenaudRohlinger** [comment]
Should be fixed with create-r3f-app 1.6.7.

Thanks for the feedbacks!
